### DISCUSSION TITLE
can anybody recommend me a mod that shows the whole location map?

### DISCUSSION THREAD

**Discussion by Kreymox** [FakeFishGames/Barotrauma#15190] [Code questions and modding support]
i've been sticking in many locations for too long, i want to get a possibility to check and fix my route to prevent wasting time
upd. could be caused by the bug that appears as a non-finishable route (sub just doesn't fit into the only way possible), won't know until i'll check that with any mod

**Comment by SomeRandomNoobKekeke** [comment]
You mean world map, not level map, right?
I don't think that exists, and i'm too lazy to make one
If you don't mind enabling cheats you can just "debugai" to see the whole map
UPD: Ye, i think you can just enablecheats, debugai - check map - and quit without save, cheat status won't be saved, probably :baromodder: 

> could be caused by the bug that appears as a non-finishable route

Very unlikely, it's [~0.0038%](https://github.com/FakeFishGames/Barotrauma/discussions/13791#discussioncomment-9173258) that you'll get one of those seeds

Also, since you're interested in "prevent wasting time", may i recomend [this](https://steamcommunity.com/sharedfiles/filedetails/?id=3299673932)?

**Reply by Kreymox** [reply]
yeah, thanks
i was talking about such map as you have sent me previously (the ruins one)
btw i was just about to write upd2 - i just fucked up, that's not a bugged seed
advice for me or the other players with the same problem - just break all the ice spires if you can't find a way forward
